### PR TITLE
Update config for AKS s2s

### PR DESCRIPTION
**Pull request by jackmaloney**

### CODE REVIEW

File: src/main/resources/bootstrap.yaml [hunk 1]
```diff
@@ -12,11 +12,10 @@ spring:
         idam-sscs-oauth2-client-secret: idam.oauth2.client.secret
         idam-oauth-user: idam.oauth2.client.id
         idam-redirect: idam.oauth2.redirectUrl
-        idam-s2s-api: idam.s2s-auth.url
-        sscs-s2s-secret: idam.s2s-auth.totp_secret
-        s2s-micro: idam.s2s-auth.microservice
+        sscs-s2s-secret: IDAM_S2S_AUTH_TOTP_SECRET
+        s2s-micro: IDAM_S2S_AUTH_MICROSERVICE
```
Comment (timja) [inline src/main/resources/bootstrap.yaml]: I would remove this, it'll always be sscs i assume?

File: infrastructure/main.tf [hunk 2]
```diff
@@ -123,9 +118,9 @@ module "track-your-appeal-notifications" {
     IDAM_URL = "${data.azurerm_key_vault_secret.idam-api.value}"
     IDAM_API_JWK_URL = "${data.azurerm_key_vault_secret.idam-api.value}/jwks"
 
-    IDAM.S2S-AUTH.TOTP_SECRET  = "${data.azurerm_key_vault_secret.sscs-s2s-secret.value}"
-    IDAM.S2S-AUTH              = "${local.s2sCnpUrl}"
-    IDAM.S2S-AUTH.MICROSERVICE = "${var.idam_s2s_auth_microservice}"
+    IDAM_S2S_AUTH_TOTP_SECRET  = "${data.azurerm_key_vault_secret.sscs-s2s-secret.value}"
+    IDAM_S2S_AUTH              = "${local.s2sCnpUrl}"
+    IDAM_S2S_AUTH_MICROSERVICE = "${var.idam_s2s_auth_microservice}"
```
Comment (timja) [inline infrastructure/main.tf]: I would remove this assuming it's always sscs and just let your default config pick it up

Comment (jackmaloney) [reply infrastructure/main.tf]: Addressed

File: src/main/resources/bootstrap.yaml (hunk 1 above)
Comment (jackmaloney) [reply src/main/resources/bootstrap.yaml]: Addressed

**Pull request merged**